docs/Community/Governance.md

```diff
@@ -0,0 +1,137 @@
+* **Open Community**: We are welcoming and respectful of everyone.
+
+
+Code of Conduct
+---------------
+
+The Open Enclave SDK community abides by the
+[OE SDK Code of Conduct](conduct/CodeOfConduct.md). Here is an
+excerpt:
+
+*In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and our
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and
+expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.*
+
+As a member of the OE SDK project, you represent the project and your fellow
+contributors. We value our community trememdously and want to build and maintain
+a friendly, inclusive, collaborative environment for all our contributors and
+users.
+
+Community Governance Structure
+------------------------------
+
+*Note(Aeva) section added*
+
+The Open Enclave SDK community uses a
+[concensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+approach to governance at all levels. To support a growing project, we have
+adopted a loose heirarchy wherein domain-specific decisions are handled within
+each domain, or Special Interest Group (**SIG**), cross-domain work is enabled
+through Working Groups (**WG**), and a Community Governance Committee (**CGC**)
+meets to review the overall structure, ensuring the health of the project.
+
+Each group (SIGs, WGs, and CGC) maintain documentation in this repository.
+Creating a new SIG or WG requires the approval of the CGC. SIGs are generally
+responsible for code, whereas a WG is not.
+
+SIG leaders are responsible for the code within their domain, and should be
+considered authoritative for that subsection of the project's code base insofar
+as code reviews and architectural discussions. When a change affects code owned
+by multiple SIGs, it should be coordinated with the respective SIG leads.
+Cross-SIG efforts can be coordinated by establishing a Working Group (WG), which
+is given a charter and which reports progress back to the relevant SIG leaders
+or the CGC.
+
+If there is conflict, either within or between SIGs, or between any members of
+the community, wherein it is desirable, the issue can be raised to the CGC.
+
+Technical Discussions
+---------------------
+
+*Note(Aeva) section added*
+
+Technical discussions generally happen in one of these places:
+
+* [on the GitHub project](https://github.com/openenclave/openenclave), either in an Issue or a Pull Request
+* in public SIG or WG meetings, which are typically held on Zoom (public calendar TBD)(TODO)
+* on our mailing list or chat server (details TBD) (TODO)
+
+While we strive for openness and transparency, it is natural that some
+discussions will also happen in person (eg, in the hallway track) or other
```

Comment:
"eg," -> "e.g.,"

but what is "the hallway track" referring to?   OE doesn't have f2f meetings today or "tracks".

Reply:
I think this is just using the conference term "hallway track" as an analogy. Some people that work on the SDK inevitably work in close physical proximity and not all discussions will happen online. Given that reality, I think this is good guidance in terms of decision-making for the SDK.

This is language that is pretty specific to people who attend a lot of conferences (I hear it most often used by academics) so I agree that language could be a little confusing.

Reply:
Jordan, your interpretation is correct. Even without a conference / event, before COVID there were random f2f discussions that happened between some folks who work in the same office. This is human nature and not something I want to discourage -- it is, rather, something I want to include and support in the process by clearly asking folks to share the outcome of any such discussions.

That said, I thank you for calling out language that isn't inclusive. I try, but still sometimes forget that I'm speaking in culturally-specific jargon some times.

I'll add clarify my use of the phrase "hallway track"